src/main/resources/com/cloudbees/jenkins/plugins/sshagent/SSHAgentStep/help.html

```diff
@@ -0,0 +1,7 @@
+<pre><code>
+node {
+  sshagent ([ignoreMissing: false, credentials: ['14714940-3b66-48f1-bf60-7ec8ec7c5581']]) {
```

Comment:
Fine but you should have a `config.jelly`. Otherwise the _Snippet Generator_ will not work.

Also you should advertise use of credentials with a mnemonic `id`.